www/js/app.js

```diff
@@ -386,6 +397,36 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         }, 600);
     });
 
+    //focus search bar (#prefix) if Home key is pressed
+    function focusPrefixOnHomeKey(event) {
+        //check if home key is pressed
+        if (event.key === 'Home') {
+            //scroll to top of #search-article section if scroll bar is associated with it
+            $('#search-article').scrollTop(0);
+            //in case the scroll bar is associated with iframe #articleContent
+            var iframeArticleDocument = document.getElementById('articleContent').contentWindow.document;
```

Comment:
@Hrishi-Raj  I think you can delete this line (407), and change the next line to:

`document.getElementById('articleContent').contentWindow.scrollTo(0, 0);

Even better, use smooth scrolling for modern browsers (not IE11 or Edge Legacy):

`document.getElementById('articleContent').contentWindow.scrollTo({top: 0, behavior: 'smooth'});`
(syntax can still be used for old browsers, it doesn't throw an error).      
`